testsuites/store/src/main/java/org/eclipse/rdf4j/sail/RDFStoreTest.java

```diff
@@ -718,6 +726,7 @@ public void testQueryBindings()
 	}
 
 	@Test
+	@Ignore("test is fundamentelly flawed as it uses multithreaded access on the same connection")
```

Comment:
An alternative to ignoring the test is fixing it, by putting the loop that checks if a txn is active (and the `con.begin()` call) inside a synchronized block that locks on the connection. That would probably fix the test failing, but on the other hand I'm not sure what this test is even testing for - we simply do not guarantee multi-threaded access on a single connection. Thoughts?